ui/app/src/components/BrowseFilesDialog/BrowseFilesDialog.tsx

```diff
@@ -31,6 +31,8 @@ export function BrowseFilesDialog(props: BrowseFilesDialogProps) {
     numOfLoaderItems,
     allowUpload,
     initialParameters,
+    preselectedPaths,
+    disableChangePreselected,
```

Comment:
Initialise the prop to false when missing (in both components)

Reply:
we were initializing it to true https://github.com/craftercms/studio-ui/pull/4227#discussion_r1846102935. Is this only for Browse? Also, we want that set to true in the controls right?